cli/version_check.go

```diff
@@ -33,6 +34,7 @@ func PopulateTerraformVersion(terragruntOptions *options.TerragruntOptions) erro
 	}
 
 	terragruntOptions.TerraformVersion = terraformVersion
+	util.Debugf(util.CreateLogger(""), "Terragrunt Version: %s", terraformVersion)
```

Comment:
This is printing the Terraform version, not the Terragrunt version. Which one did you actually want to print?

Reply:
> This is printing the Terraform version, not the Terragrunt version. Which one did you actually want to print?

My bad, I actually meant `Terraform Version: %s", terraformVersion`.